via/services/url_details.py

```diff
@@ -0,0 +1,59 @@
+"""Retrieve details about a resource at a URL."""
+import cgi
+from collections import OrderedDict
+
+from via.requests_tools.headers import add_request_headers, clean_headers
+from via.services import HTTPService
+from via.services.google_drive import GoogleDriveAPI
+from via.services.youtube import YoutubeService
+
+
+class URLDetailsService:
```

Comment:
Made this into a service.

This was a function which already had a dependency on HTTPService and I was about to add one for YoutubeService.

Made into a service in one commit and extended with the YoutubeService dependency next.